mlflow/genai/judges/adapters/databricks_adapter.py

```diff
@@ -0,0 +1,439 @@
+
+    @context.eval_context
+    def _call_chat_completions(user_prompt: str, system_prompt: str):
+        managed_rag_client = context.get_context().build_managed_rag_client()
+
+        return managed_rag_client.get_chat_completions_result(
+            user_prompt=user_prompt,
+            system_prompt=system_prompt,
+        )
+
+    return _call_chat_completions(user_prompt, system_prompt)
+
+
+def _parse_databricks_judge_response(
+    llm_output: str | None,
+    assessment_name: str,
+) -> Feedback:
+    """
+    Parse the response from Databricks judge into a Feedback object.
+
+    Args:
+        llm_output: Raw output from the LLM, or None if no response.
+        assessment_name: Name of the assessment.
+
+    Returns:
+        Feedback object with parsed results or error.
+    """
+    source = AssessmentSource(
+        source_type=AssessmentSourceType.LLM_JUDGE, source_id=_DATABRICKS_DEFAULT_JUDGE_MODEL
+    )
+    if not llm_output:
+        return Feedback(
+            name=assessment_name,
+            error="Empty response from Databricks judge",
+            source=source,
+        )
+    try:
+        response_data = json.loads(llm_output)
+    except json.JSONDecodeError as e:
+        return Feedback(
+            name=assessment_name,
+            error=f"Invalid JSON response from Databricks judge: {e}",
+            source=source,
+        )
+    if "result" not in response_data:
+        return Feedback(
+            name=assessment_name,
+            error=f"Response missing 'result' field: {response_data}",
+            source=source,
+        )
+    return Feedback(
+        name=assessment_name,
+        value=response_data["result"],
+        rationale=response_data.get("rationale", ""),
+        source=source,
+    )
+
+
+def _invoke_databricks_judge(
+    prompt: str | list["ChatMessage"],
+    assessment_name: str,
+) -> Feedback:
+    """
+    Invoke the Databricks judge using the databricks.agents.evals library.
+
+    Uses the direct chat completions API for clean prompt submission without
+    any additional formatting or template requirements.
+
+    Args:
+        prompt: The formatted prompt with template variables filled in.
+        assessment_name: The name of the assessment.
+
+    Returns:
+        Feedback object from the Databricks judge.
+
+    Raises:
+        MlflowException: If databricks-agents is not installed.
+    """
+    try:
+        if isinstance(prompt, str):
+            system_prompt = None
+            user_prompt = prompt
+        else:
+            prompts = _split_messages_for_databricks(prompt)
+            system_prompt = prompts.system_prompt
+            user_prompt = prompts.user_prompt
+
+        llm_result = call_chat_completions(user_prompt, system_prompt)
+        return _parse_databricks_judge_response(llm_result.output, assessment_name)
+
+    except Exception as e:
+        _logger.debug(f"Failed to invoke Databricks judge: {e}")
```

Comment:
```suggestion
        _logger.debug(f"Failed to invoke Databricks judge: {e}", exc_info=True)
```

nit: Can we display stack trace in the debug logging?